### ISSUE TITLE
Investigate Buffer for on-boarding package

### ISSUE THREAD

**Issue by seejessicacode** [techlahoma/user-groups#5]
Some questions we need to answer:
- Can we grant user group leaders permission to connect to their social media account or do they have to give their account credentials to us in order to add their accounts to Buffer?
- Is there an option to auto share/retweet user group posts/tweets from Techlahoma accounts? Do we want to do so without reviewing content?
- What is the cost and how does it scale per user group social media account?
- Are there any options for including or reacting to Meetup events?

Suggestions for any other questions to consider are welcome!

**Comment by seejessicacode** [comment]
I was able to add members to manage/contribute to select social profiles.
<img width="437" alt="buffrshareaccess" src="https://cloud.githubusercontent.com/assets/943398/18497763/d5f48476-79f6-11e6-85a1-b1e0f5e206fe.PNG">

However, those members cannot add their group social media to the Techlahoma Buffr account.

**TL;DR:** UG leaders will have to share credentials for their group social media accounts with Techlahoma in order for them to take advantage of scheduling and/or approving content for publishing.
